File: SWIG-C++/example_main.lua
#!/usr/bin/luajit

require ("cpp_example")

function main()
    a = cpp_example.Example('teszt-A')
    b = cpp_example.Example('teszt-B')

    print (a:Get())
    print (b:Get())
    print (a:Get())
    print (a:Get())
    print (a:Get())
    print (a:Get())
    print (b:Get())
end

main()

File: SWIG-C++/example_main.cs
using System;

public class Example_main
{
    public static void Main(string[] args)
    {
        Example a = new Example("Teszt-A");
        Example b = new Example("Teszt-B");

        Console.WriteLine(a.Get());
        Console.WriteLine(b.Get());
        Console.WriteLine(a.Get());
        Console.WriteLine(a.Get());
        Console.WriteLine(a.Get());
        Console.WriteLine(a.Get());
        Console.WriteLine(b.Get());
    }
}
